### ISSUE TITLE
There is a chance that cloud side communication will fail

### ISSUE THREAD

**Issue by lgy1027** [kubeedge/edgemesh#244]
Half the chance of success in cloud communication test
https://edgemesh.netlify.app/zh/guide/test-case.html#%E8%B7%A8%E8%BE%B9%E4%BA%91%E9%80%9A%E4%BF%A1

Cloud access edge
````
/ # telnet tcp-echo-edge-svc.edgezone 2701
Welcome, you are connected to node edge.
Running on Pod tcp-echo-edge-54dcccbf6f-7254p.
In namespace edgezone.
With IP address 172.17.0.5.
Service default.
dd
dd
/ # telnet tcp-echo-edge-svc.edgezone 2701
Connection closed by foreign host
/ # telnet tcp-echo-edge-svc.edgezone 2701
Welcome, you are connected to node edge.
Running on Pod tcp-echo-edge-54dcccbf6f-7254p.
In namespace edgezone.
With IP address 172.17.0.5.
Service default.
/ # telnet tcp-echo-edge-svc.edgezone 2701
Connection closed by foreign host
/ # telnet tcp-echo-edge-svc.edgezone 2701
Welcome, you are connected to node edge.
Running on Pod tcp-echo-edge-54dcccbf6f-7254p.
In namespace edgezone.
With IP address 172.17.0.5.
Service default.
/ # telnet tcp-echo-edge-svc.edgezone 2701
Connection closed by foreign host
/ # 
````
agent log:
````
I1214 06:46:07.687102       1 tcpproxy.go:42] Get a new stream from QmcFrmzuBMxnT7t3RbGh7aKSYE9m7NGJujsxPukfXQj9nr
I1214 06:46:07.687209       1 tcpproxy.go:61] l4 proxy get tcp server address: 172.17.0.5:2701
I1214 06:46:10.670957       1 tcpproxy.go:105] Success proxy for targetAddr: 172.17.0.5:2701
I1214 06:46:12.466012       1 tcpproxy.go:42] Get a new stream from QmcFrmzuBMxnT7t3RbGh7aKSYE9m7NGJujsxPukfXQj9nr
I1214 06:46:12.466035       1 tcpproxy.go:61] l4 proxy get tcp server address: 172.17.0.4:2701
E1214 06:46:12.466404       1 tcpproxy.go:71] l4 proxy connect to 172.17.0.4:2701 err: dial tcp 172.17.0.4:2701: connect: connection refused
I1214 06:46:58.359288       1 tcpproxy.go:42] Get a new stream from QmcFrmzuBMxnT7t3RbGh7aKSYE9m7NGJujsxPukfXQj9nr
I1214 06:46:58.359344       1 tcpproxy.go:61] l4 proxy get tcp server address: 172.17.0.5:2701
I1214 06:47:00.258184       1 tcpproxy.go:105] Success proxy for targetAddr: 172.17.0.5:2701
I1214 06:47:01.393185       1 tcpproxy.go:42] Get a new stream from QmcFrmzuBMxnT7t3RbGh7aKSYE9m7NGJujsxPukfXQj9nr
I1214 06:47:01.393240       1 tcpproxy.go:61] l4 proxy get tcp server address: 172.17.0.4:2701
E1214 06:47:01.393895       1 tcpproxy.go:71] l4 proxy connect to 172.17.0.4:2701 err: dial tcp 172.17.0.4:2701: connect: connection refused
I1214 06:47:10.295143       1 tcpproxy.go:42] Get a new stream from QmcFrmzuBMxnT7t3RbGh7aKSYE9m7NGJujsxPukfXQj9nr
I1214 06:47:10.295176       1 tcpproxy.go:61] l4 proxy get tcp server address: 172.17.0.5:2701
I1214 06:47:11.791694       1 tcpproxy.go:105] Success proxy for targetAddr: 172.17.0.5:2701
I1214 06:47:13.233480       1 tcpproxy.go:42] Get a new stream from QmcFrmzuBMxnT7t3RbGh7aKSYE9m7NGJujsxPukfXQj9nr
I1214 06:47:13.233510       1 tcpproxy.go:61] l4 proxy get tcp server address: 172.17.0.4:2701
E1214 06:47:13.234160       1 tcpproxy.go:71] l4 proxy connect to 172.17.0.4:2701 err: dial tcp 172.17.0.4:2701: connect: connection refused
````

Edge access cloud
````
/ # 
/ # telnet tcp-echo-cloud-svc.cloudzone 2701
Welcome, you are connected to node master.
Running on Pod tcp-echo-cloud-6487cd44d5-p5g28.
In namespace cloudzone.
With IP address 10.233.70.42.
Service default.
/ # telnet tcp-echo-cloud-svc.cloudzone 2701
Connection closed by foreign host
/ # telnet tcp-echo-cloud-svc.cloudzone 2701
Welcome, you are connected to node master.
Running on Pod tcp-echo-cloud-6487cd44d5-p5g28.
In namespace cloudzone.
With IP address 10.233.70.42.
Service default.
/ # telnet tcp-echo-cloud-svc.cloudzone 2701
Connection closed by foreign host
````
Log
````
I1214 06:49:49.263813       1 dns.go:55] dns server parse tcp-echo-cloud-svc.cloudzone ip 10.233.6.181
I1214 06:49:49.263980       1 proxy.go:38] clusterIP: 10.233.6.181, servicePort: 2701
W1214 06:49:49.264069       1 util.go:21] DestinationRule "cloudzone.tcp-echo-cloud-svc" not found, use default strategy [RoundRobin] from config file
I1214 06:49:49.264079       1 log.go:181] DEBUG: add [2] handlers for chain [tcp]
I1214 06:49:49.264122       1 handler.go:59] l4 proxy get tcpserver address: master:10.233.70.42:2701
I1214 06:49:49.265370       1 tcpproxy.go:166] master dial 10.233.70.42:2701 success
I1214 06:49:49.265396       1 handler.go:106] l4 proxy start proxy data between tcpserver master:10.233.70.42:2701
I1214 06:50:06.015259       1 handler.go:120] Success proxy to master:10.233.70.42:2701
I1214 06:50:07.856117       1 dns.go:55] dns server parse tcp-echo-cloud-svc.cloudzone ip 10.233.6.181
I1214 06:50:07.856346       1 proxy.go:38] clusterIP: 10.233.6.181, servicePort: 2701
W1214 06:50:07.856428       1 util.go:21] DestinationRule "cloudzone.tcp-echo-cloud-svc" not found, use default strategy [RoundRobin] from config file
I1214 06:50:07.856522       1 log.go:181] DEBUG: add [2] handlers for chain [tcp]
I1214 06:50:07.856555       1 handler.go:59] l4 proxy get tcpserver address: master:10.233.70.43:2701
````

**Comment by Poorunga** [comment]
yep, this example has some problem, we are troubleshooting.

**Comment by Poorunga** [comment]
fix in #333 

/close
